File: 프로그래머스/0/120802. 두 수의 합/두 수의 합.js
function solution(num1, num2) {
    var answer = -1;
    var max =50000;
    var min = -50000;
    if(num1<=50000 || num1>=-5000&&num2<=50000 || num2>=-5000){
        answer = num1+num2;
    }
    return answer;
}

File: 프로그래머스/0/120802. 두 수의 합/두 수의 합.java
class Solution {
    public int solution(int num1, int num2) {
        int Max = 50000;
        int min = -50000;
        int answer = -1;
            
            if(num1>= -50000 || num1<=50000 && num2>= -50000 || num2<=50000 ){
                answer = num1+num2;
            }
        return answer;
    }
}